packages/veritone-react-common/package.json

```diff
@@ -19,7 +19,9 @@
     "react-dnd-test-backend": "^2.5.4",
     "react-dom": "^16.2.0",
     "react-infinite-calendar": "^2.3.1",
+    "react-lines-ellipsis": "^0.10.2",
```

Comment:
I added a different package for multiline ellipsis in the engine select PR. I believe I attempted to use `react-lines-ellipsis` but ran into some issues with custom fonts. Could we reuse the same one please ? https://github.com/CezaryDanielNowak/React-dotdotdot

Reply:
fixed